### ISSUE TITLE
Pytest warning on Datastore test

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nucypher#2205]
```
tests/unit/datastore/test_datastore.py:28
  /home/circleci/nucypher/tests/unit/datastore/test_datastore.py:28: PytestCollectionWarning: cannot collect test class 'TestRecord' because it has a __init__ constructor (from: datastore/test_datastore.py)
    class TestRecord(DatastoreRecord):
```

**Comment by KPrasch** [comment]
This can be resolved by adding the class attribute `    __test__ = False # prohibit Pytest from picking it up`

**Comment by cygnusv** [comment]
Closed by 828a630c0ec2481ed095aa91c3c95a4609c157ad
